girder/api/docs.py

```diff
@@ -112,13 +112,25 @@ def removeRouteDocs(resource, route, method, info, handler):
                 del routes[resource]
 
 
-def addModel(name, model):
+def addModel(resource, name, model):
     """
-    This is called to add a model to the swagger documentation.
+    Add a model to the Swagger documentation.
 
+    :param resource: The type of existing resource to add the model to.
+    :param resource: str
     :param name: The name of the model.
     :type name: str
     :param model: The model to add.
     :type model: dict
```

Comment:
@cjh1 I would have no problem with a convenience method to add the same model to multiple resources at once, though it's not hard to just call the function a few times, since the actual model dict will be only be shallow-copied.

I can't think of a case where you'd add a model to every single Girder resource, including the built-in ones (what will an Assetstore, Group, and Notification have in common?). I am open to a solid use case to change my mind here.